File: 第一期/S1T182/src/class11/Student.java
package class11;

public class Student {

	//属性：
	int stuId ;		//学号
	String stuName;	//姓名
	String stuSex;	//性别
	int stuAge;		//年龄
	int classId;	//所属班级编号
	
	//方法：
	/**
	 * 显示学生信息的方法
	 */
	public String toString(){
		
		return stuId + " " + stuName + " " + stuSex + " " + stuAge + " " + classId ;
		
	}

}

File: 第二期/Example/Example4/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Example4
{
    public enum Sex
    {
        男,女
    }

    /// <summary>
    /// 学生类
    /// </summary>
    public class Student
    {
        #region 字段
        //学号
        private string stuNO;

        //姓名
        private string stuName;

        //性别
        private Sex stuSex;

        //年龄
        private int stuAge;

        //地址
        private string stuAddress;
        #endregion

        #region 属性
        //学号
        public string StuNO
        {
            get { return stuNO; }
            set { stuNO = value; }
        }

        //姓名
        public string StuName
        {
            get { return stuName; }
            set { stuName = value; }
        }

        //性别
        public Sex StuSex
        {
            get { return stuSex; }
            set { stuSex = value; }
        }

        //年龄
        public int StuAge
        {
            get { return stuAge; }
            set { stuAge = value; }
        }

        //地址
        public string StuAddress
        {
            get { return stuAddress; }
            set { stuAddress = value; }
        }
        #endregion

        #region 方法

        #endregion
    }
}
